### PR TITLE
feat(#72): Replace hardcoded values with env variables

### LINKED ISSUE #72: Replace harcoded values with environment variables in root.yml
Database variables are currently hardcoded in `models/root/root.yml` and `models/root/couchdb.sql`. 

They need to be replaced with environment variables.

### PR DESCRIPTION
**Pull request by andrablaj**
Closes #72

### CODE REVIEW

**Comment by andrablaj** [comment]
@lorerod @njuguna-n I tried to do similar changes as you did for [BRAC Models](https://github.com/medic/cht-pipeline/commit/d7e70b72b6ae80213f16d98e49c0dc6e124f88dc). I didn't manage to get the unit tests pass, though. Do they pass for BRAC Models? 

There is still a `v1` hardcoded value in couchdb.sql that I didn't manage to replace by an env variable successfully at the moment.

**Comment by lorerod** [comment]
>  I didn't manage to get the unit tests pass, though.

[@andrablaj](https://github.com/andrablaj), the unit tests are passing on CI. Do you mean locally?

**Comment by lorerod** [comment]
@andra, hi!

> I get the same error for the main branch too when I run the tests locally.

How are you running the tests locally? 
I checked out this branch, and it is working for me locally. You must execute the three shell scripts setup every time you run the test: `setup.sh`, `run_dbt_tests.sh`, and `tear_down.sh`.

> Env var required but not provided: 'DBT_POSTGRES_USER'

This env var is setup in [this](https://github.com/medic/cht-pipeline/blob/12bbe00e115a6f8ac78c77cbb738fbc0829f19ac/tests/setup.sh#L8) line of `setup.sh`

Let me know if it works for you.

**Comment by andrablaj** [comment]
@lorerod I tried the steps in the `README,` and double-checked my env variables, and they seem to align with what's needed to run the tests. As the CI passes and both you and Tom manage to run the tests locally, can you please review the PR and share comments, if any, the time I fix my local env? Thank you!

File: packages.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 packages:
   - package: dbt-labs/dbt_utils
     version: 1.1.1
-  - package: tnightengale/dbt_meta_testing
-    version: 0.3.6
+  - package: calogica/dbt_date
```
[the remark this answers is not in the document]
Comment (andrablaj) [reply packages.yml:4]: I changed it to map[ this commit](https://github.com/medic/cht-pipeline/commit/6dc9117426799fb15323ac1369b3f81a3b62d52d) from the `BRAC-models` branch, as `main` was behind. Should I change it back?

File: models/root/couchdb.sql [hunk 2]
```diff
@@ -12,4 +10,4 @@
 SELECT
     doc->>'type' AS type,
     *
-FROM {{ import_couchdb_data }}
+FROM v1.{{ env_var('POSTGRES_TABLE') }}
```
[the remark this answers is not in the document]
Comment (andrablaj) [reply models/root/couchdb.sql:13]: As mentioned [here](https://github.com/medic/cht-pipeline/pull/73#issuecomment-1957529239), I didn't manage to get that working. I will try again tomorrow.

**Review by lorerod — CHANGES_REQUESTED** [review]
Hello @andrablaj! Thank you for making the improvements. I have a few minor suggestions to make. 

Before we merge this to the main branch, I think we need to ensure that the `cht-sync` tests pass with this version of `cht-pipeline`, right @njuguna-n?
I can check when you have a final version of this, and if the tests are not passing, I will create an issue on that repository and make sure this is mentioned as a dependency.
What do you think?

**Comment by njuguna-n** [comment]
@lorerod yes, I agree that we need to have tests passing.

**Pull request merged**